packages/fxa-auth-server/test/local/routes/utils/clients.js

```diff
@@ -108,7 +108,7 @@ describe('clientUtils.formatLocation', () => {
     assert.equal(log.warn.callCount, 0);
   });
 
-  it('unsets location if it fails', () => {
+  it.skip('unsets location if it fails', () => {
```

Comment:
Hmm yeah I'm also not sure why we would want to just throw out the entire thing if something funky happens when trying to determine the locale. It makes sense to me to just default to `'en'` since that's the language we're going to show the user if it can't be determined. 

Maybe the idea was that if that's invalid, the rest of it is probably also invalid, so throw out everything. I'm personally fine with ripping the test out, and showing the location info regardless. We also have other tests around bogus location info so I feel like we're covered. I'll tag @vbudhram here in case there's more context he may have but if we all think it should be ripped out then 🤷‍♀️